code/xcvm/cosmos/flake-parts.nix

```diff
@@ -0,0 +1,34 @@
+{ self, ... }: {
```

Comment:
Please rename this file to `cosmwasm.nix` instead of `flake-parts.nix`

Reply:
https://github.com/hercules-ci/flake-parts/tree/main/template/multi-module/hello

Reply:
renamed, hopefully according dicussion